File: src/Entity/event.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entity;
import java.sql.Date;

/**
 *
 * @author Taîga
 */
public class event {
    int id;
    String nom;
    String lieu;
    Date date;
    int capacite;
    String description;
    int capaciteinitiale;

    public event() {
    }

    public event(int id, String nom, String lieu, Date date, int capacite, String description) {
        this.id = id;
        this.nom = nom;
        this.lieu = lieu;
        this.date = date;
        this.capacite = capacite;
        this.description = description;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getLieu() {
        return lieu;
    }

    public void setLieu(String lieu) {
        this.lieu = lieu;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public int getCapacite() {
        return capacite;
    }

    public void setCapacite(int capacite) {
        this.capacite = capacite;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getCapaciteinitiale() {
        return capaciteinitiale;
    }

    public void setCapaciteinitiale(int capaciteinitiale) {
        this.capaciteinitiale = capaciteinitiale;
    }

    @Override
    public String toString() {
        return "event{" + "id=" + id + ", nom=" + nom + ", lieu=" + lieu + ", date=" + date + ", capacite=" + capacite + ", description=" + description + ", capaciteinitiale=" + capaciteinitiale + '}';
    }

    public event(int id, String nom, String lieu, Date date, int capacite, String description, int capaciteinitiale) {
        this.id = id;
        this.nom = nom;
        this.lieu = lieu;
        this.date = date;
        this.capacite = capacite;
        this.description = description;
        this.capaciteinitiale = capaciteinitiale;
    }

    
    
    
}

File: Final/src/Entity/Event.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

use Symfony\Component\Validator\Constraints as Assert;

/**
 * Event
 *
 * @ORM\Table(name="event")
 * @ORM\Entity
 */
class Event
{
    /**
     * @var int
     *
     * @ORM\Column(name="ID", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *  @Assert\NotBlank(message="champ obligatoire")
     *  @Assert\Unique(message="ce nom est déjà utiliser ")
     * @ORM\Column(name="NOM", type="string", length=99, nullable=false)
     */
    private $nom;

    /**
     * @var string
     * @Assert\Length(
     *      min = 2,
     *      max = 30,
     *      minMessage = " Minimum {{ limit }} caracteres",
     *      maxMessage = " Maximum {{ limit }} caracteres"
     * )
     * @Assert\NotBlank(message="champ obligatoire")
     *
     * @ORM\Column(name="lieu", type="string", length=99, nullable=false)
     */
    private $lieu;

    /**
     * @var \DateTime
     * @Assert\NotBlank(message="champ obligatoire")
     * @ORM\Column(name="Date", type="date", nullable=false)
     */
    private $date;

    /**
     * @var int
     * @Assert\Length(
     *      min = 10,
     *      max = 255,
     *      minMessage = " Minimum {{ limit }} caracteres",
     *      maxMessage = " Maximum {{ limit }} caracteres"
     * )
     * @Assert\NotBlank(message="champ obligatoire")
     * @ORM\Column(name="capacite", type="integer", nullable=false)
     */
    private $capacite;

    /**
     * @var string
     ** @Assert\Length(
     *      min = 10,
     *      max = 255,
     *      minMessage = " Minimum {{ limit }} caracteres",
     *      maxMessage = " Maximum {{ limit }} caracteres"
     * )
     * @ORM\Column(name="description", type="string", length=250, nullable=false)
     */
    private $description;

    /**
     * @var int
     * @Assert\Length(
     *      min = 10,
     *      max = 255,
     *      minMessage = " Minimum {{ limit }} caracteres",
     *      maxMessage = " Maximum {{ limit }} caracteres"
     * )
     * @Assert\NotBlank(message="champ obligatoire")
     * @ORM\Column(name="capaciteinitiale", type="integer", nullable=false)
     */
    private $capaciteinitiale;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNom(): ?string
    {
        return $this->nom;
    }

    public function setNom(string $nom): self
    {
        $this->nom = $nom;

        return $this;
    }

    public function getLieu(): ?string
    {
        return $this->lieu;
    }

    public function setLieu(string $lieu): self
    {
        $this->lieu = $lieu;

        return $this;
    }

    public function getDate(): ?\DateTimeInterface
    {
        return $this->date;
    }

    public function setDate(\DateTimeInterface $date): self
    {
        $this->date = $date;

        return $this;
    }

    public function getCapacite(): ?int
    {
        return $this->capacite;
    }

    public function setCapacite(int $capacite): self
    {
        $this->capacite = $capacite;

        return $this;
    }

    public function getDescription(): ?string
    {
        return $this->description;
    }

    public function setDescription(string $description): self
    {
        $this->description = $description;

        return $this;
    }

    public function getCapaciteinitiale(): ?int
    {
        return $this->capaciteinitiale;
    }

    public function setCapaciteinitiale(int $capaciteinitiale): self
    {
        $this->capaciteinitiale = $capaciteinitiale;

        return $this;
    }


}
